solidity/contracts/bridge/Wallets.sol

```diff
@@ -40,8 +40,12 @@ library Wallets {
         ///      fulfill their pending redemption requests although new
         ///      redemption requests and new deposit reveals are not accepted.
         MovingFunds,
-        /// @dev The wallet moved or redeemed all their funds and cannot
-        ///      perform any action.
+        /// @dev The wallet moved or redeemed all their funds and is in the
+        ///      closing period where they can be subject of fraud challenges
+        ///      and must defend against them.
```

Comment:
We could mention for future-us and auditors that this state is needed to protect against deposit frauds on deposits revealed but not swept and that the period in which the wallet is in a closing state must be greater that the deposit refund time plus some time margin.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/213/commits/2c05425bca213ea7220f7f8efe4cb4dd96b8ce22